tutorials/XGI in 5 minutes.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>I would put this before the circular layout, so that the user can immediately check that the plotted structure corresponds to the output of <code>H.edges.members()</code>. Also I'm wondering if we should use fixed seed to keep the same layout the three times that we use <code>xgi.draw()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>